src/main/java/io/github/gaming32/niceload/client/NiceLoadMod.java

```diff
@@ -23,13 +31,24 @@ public NiceLoadMod() {
     @Override
     public void onPreLaunch() {
         LOGGER.info("Early bird gets the worm!");
+
+        AutoConfig.register(NiceLoadConfig.class, Toml4jConfigSerializer::new);
+        NiceLoadConfig config = AutoConfig.getConfigHolder(NiceLoadConfig.class).getConfig();
+
+        SplashOverlay.MOJANG_RED = ColorHelper.Argb.getArgb(config.backgroundColor[3], config.backgroundColor[0], config.backgroundColor[1], config.backgroundColor[2]);
```

Comment:
Minecraft classes should not be initialized in preLaunch. Additionally, overwriting the color could cause incompatibilities with other mods doing the same. I feel like it isn't NiceLoad's job to change the color of anything it doesn't add.

Reply:
I think it's okay. The idea for this PR was to add configurability so that the user or modpack dev can change the colors in the splash screen, similar to the way forge used to handle the `splash.properties` file.

Reply:
It's getting removed anyway.